daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-patterns.md

```diff
@@ -56,14 +56,61 @@ catch (TaskFailedException) // Task failures are surfaced as TaskFailedException
 }
 ```
 
+{{% alert title="Note" color="primary" %}}
```

Comment:
Should this note also be in the Python example? It has the same brevity?

Reply:
For Python, the code was concise enough that I went ahead and included everything - thus I moved this note to be just part of the .NET tab.